content/sql/concepts/dates/terms/between/between.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'BETWEEN'
+Description: 'Returns records where a specified value is within a given range, including the boundary values.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Date'
+  - 'Queries'
+  - 'SQL Server'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`BETWEEN`** [operator](https://www.codecademy.com/resources/docs/sql/operators) is used to find values within a given range. The values can be texts, dates, or numbers. Also, this operator is _inclusive_, i.e., both beginning and ending values are included.
+
+## Syntax
+
+```pseudo
+SELECT column_name(s) from table_name where column_name between value1 and value2;
+```
+
+> Note: The basic syntax for `BETWEEN` is generally same across most databases, but some differences may arise depending on how each database handles specific [data types](https://www.codecademy.com/resources/docs/sql/data-types) or expressions. To use this operator on a particular database, concepts such as _data handling_, _null handling_, _case sensitivity_, and _collations_ need to be checked first to avoid errors.
```

Comment:
```suggestion
> Note: The basic syntax for `BETWEEN` is generally the same across most databases, but some differences may arise depending on how each database handles specific [data types](https://www.codecademy.com/resources/docs/sql/data-types) or expressions. To use this operator on a particular database, data handling, null handling, case sensitivity, and collations must be checked first to avoid errors.
```